annotation-processor/src/main/resources/org/hibernate/validator/ap/ValidationProcessorMessages.properties

```diff
@@ -38,3 +38,5 @@ INVALID_PATTERN_ANNOTATION_PARAMETERS=Invalid annotation parameters. Regular exp
 INVALID_SCRIPT_ASSERT_ANNOTATION_PARAMETERS=Invalid annotation parameters. Script, lang and alias parameters cannot be blank.
 INVALID_DIGITS_ANNOTATION_PARAMETERS=Invalid annotation parameters. Fraction and integer should be greater than or equal to 0.
 INVALID_DECIMAL_MIN_MAX_ANNOTATION_PARAMETERS=Invalid annotation parameters. Value should be a valid BigDecimal string representation.
+INCORRECT_METHOD_PARAMETERS_OVERRIDING=Method is incorrectly overridden. Method parameters cannot have a wider range of constraints than the one from a parent class.
+INCORRECT_METHOD_RETURN_OVERRIDING=Method is incorrectly overridden. Overridden method return value cannot have wider range of constraints than the overriding one.
```

Comment:
Same as above with "Return value constraints of {name of parent class} must not be weakened in sub type."